src/components/Menu.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <oc-application-menu name="coreMenu" v-model="sidebarIsVisible" @close="sidebarIsVisible = false">
+  <oc-application-menu name="coreMenu" v-model="sidebarIsVisible" @close="sidebarIsVisible = false" close-button-label="closeButtonLabel">
```

Comment:
Is this working for you? Should be 
```vuejs
:close-button-label="closeButtonLabel"

Reply:
It was, but still not the thing I originally intended. [Fixed](https://github.com/owncloud/phoenix/commit/1894bfc9b725d28bfab76ca9f72a94006882ec77).